tests/expr/t08_if_only_body_void.test

```diff
@@ -0,0 +1,8 @@
+discard """
+  description: "an if-then with void typed body is still a unit"
+"""
+(Module
+  (ProcDecl (Ident "a") (UnitTy) (Params)
+    (Return (If (Ident "true") (Unreachable))))
+  (ProcDecl (Ident "main") (UnitTy) (Params)
+    (Return (Call (Ident "a")))))
```

Comment:
```suggestion
  description: "An if-then with void typed body is still a unit"
  output: "(): unit"
"""
(If (Ident "false") (Unreachable))
```
This makes the test more narrow, removing dependencies on other language features (i.e., procedures and `Return`).